myapp/app/views/labels/_form.html.erb

```diff
@@ -11,12 +11,20 @@
     </div>
   <% end %>
 
-  <div class="field">
-    <%= form.label :name, t('attributes.model.task.labelname') %>
-    <%= form.text_field :name, autofocus: true  %>
-  </div>
+  <div class="container" style="width: 80%; margin: 0 auto;">
+  <div class="row">
+    <div class="col-sm-12">
+  <div class="form-group p-3 mb-2 bg-light text-dark border border-secondary">
+      <%= form.label :name, t('attributes.model.task.labelname') %>
+      <%= form.text_field :name, class: 'form-control' %>
+    </div>
+    </div>
+    </div>
+    </div>  
```

Comment:
It would be better to indent HTML code to let other members know which opening tags are paired with closing tags. Most editors have ability to automatically indent the code.

Reply:
Sorry about this but these identations in my local files are correct I dont know why they were not reflected in git 😕